### PR TITLE
Artwork Endpoint

### PR DESCRIPTION
**Pull request by Micalobia**
Added the artwork endpoint, as well as support for thumbsize to work on fake_image_url, if it's there.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -319,9 +319,13 @@ def call_cargo(parameters, request_args): # Request args are passed in just for
...
                     r = requests.get(BASE_URL_WIKI + 'Special:FilePath/' + item['image_url'].rsplit('/', 1)[-1] + '?width=' + request.args.get('thumbsize'))
                     item['image_url'] = r.url
+                    # If this is a painting that has a fake, fetch that too
+                    if item.get('fake','0')=='1':
+                        r = requests.get(BASE_URL_WIKI + 'Special:FilePath/' + item['fake_image_url'].rsplit('/', 1)[-1] + '?width=' + request.args.get('thumbsize'))
+                        item['fake_image_url'] = r.url
                 except:
                     abort(500, description=error_response("Error while getting image CDN thumbnail URL.", "Failure occured with the following parameters: {}.".format(parameters)))
-
+
```
Comment (KevinPayravi) [inline app.py:328]: Small lint issue, extra spaces on this new line

**Review by KevinPayravi — CHANGES_REQUESTED** [review]

**Comment by Micalobia** [comment]
Okay, so I've done the following:.
- Removed the `image` field from art.
- Added the `_pageName=url` field to art.
- Changed the `fake` field in art to `has_fake`.
- Changed the `?fake=bool` query in the `/nh/art` endpoint to `?hasfake-bool`.
- Renamed the `get_art` and `get_art_all` functions to `get_nh_art` and `get_nh_art_all` to comply with similar functions.
- Fixed the linting issue on line 328.

**Comment by Micalobia** [comment]
Forgot docs; On that now

**Comment by Micalobia** [comment]
Should be good to go, long as I didn't miss anything

**Review by KevinPayravi — APPROVED** [review]
LGTM

**Comment by KevinPayravi** [comment]
Great job setting this up, and thanks for the contribution!

**Pull request merged**